main.js

```diff
@@ -0,0 +1,149 @@
+//Unit 1
+const multiplicationTable = (x) => {
+  for (let i = 1; i <= 10; i++)
+  console.log(x + 'x' + i + '=' + (x * i))
+}
+
+//Unit 2
+function printEvenNumbers(n) {
```

Comment:
```suggestion
function printEvenNumbers (n) {
```
Khi khai báo hàm thì thêm 1 dấu cách trước `(`